packages/messaging-system/src/components/card/card.jsx

```diff
@@ -0,0 +1,43 @@
+// External dependencies
+import React from 'react';
+import PropTypes from 'prop-types';
+// Internal dependencies
+import styles from './card.module.scss';
+
+export class Card extends React.PureComponent {
```

Comment:
Can you refactor this to be a function based component instead of a class-based component?

Reply:
I don't think so, due to we are using es2021 in eslint

Reply:
I don't think that will cause issues

Reply:
try this:
```javascript
// External dependencies
import React from 'react';
import PropTypes from 'prop-types';

// Internal dependencies
import styles from './card.module.scss';

function Card({
  username,
  lastMessage,
  profilePicture,
}) {
  return (
    <section className={styles.container}>
      <div className={styles.chatCard}>
        <figure>
          <img
            src={profilePicture}
            alt="Profile"
          />
        </figure>
        <ul className={styles.info}>
          <li className={styles.infoUsername}>{username}</li>
          <li className={styles.infoMessage}>
            {lastMessage}
          </li>
        </ul>
      </div>
    </section>
  );
}

Card.propTypes = {
  username: PropTypes.string.isRequired,
  lastMessage: PropTypes.string.isRequired,
  profilePicture: PropTypes.string.isRequired,
};

export default Card;
```